zstd/framedec.go

```diff
@@ -39,6 +39,9 @@ type frameDec struct {
 
 	rawInput byteBuffer
 
+	// Optional byte buffer that can be reused.
+	bBuf *byteBuf
```

Comment:
It would be better to use `bBuf byteBuf` or even `bBuf []byte`. This looks more clear and it is simpler to use